mne/viz/evoked.py

```diff
@@ -585,9 +585,9 @@ def _plot_image(data, ax, this_type, picks, cmap, unit, units, scalings, times,
         vmin=vmin, vmax=vmax, mask_style=mask_style, mask_alpha=mask_alpha,
         mask_cmap=mask_cmap)
 
+    # ignore xlim='tight'; happens automatically with `extent` in imshow
+    xlim = None if xlim == 'tight' else xlim
     if xlim is not None:
-        if xlim == 'tight':
-            xlim = (times[0], times[-1])
```

Comment:
Before if `xlim='tight'` then it would `ax.set_xlim((times[0], times[-1]))`. It no longer does this. But that's okay because `_plot_masked_image` sets the xlim to what `tight` would be anyway, right?

Reply:
exactly. `tight` is still useful when plot_type != 'image'.  But here, the way we're using `imshow(... extent=whatever)` in `_plot_masked_image` takes care of what `tight` is supposed to do.

Reply:
the comment on line 588 was meant to communicate that; should I clarify it?

Reply:
No I just totally missed it